apps/prairielearn/assets/scripts/instructorQuestionSettingsClient.ts

```diff
@@ -1,6 +0,0 @@
-import { onDocumentReady } from '@prairielearn/browser-utils';
-import './lib/changeIdButton.js';
```

Comment:
Deleting this file is removing functionality from this page, mainly the controls of the "change ID" button.

Reply:
Good catch, restored in 6420857.